src/python/controller/model_builder.py

```
@@ -398,6 +426,33 @@ def __fill_model_file(_model_file: ModelFile,
+            # root is Validating if it has a validate status, is in an expected state, and exists locally
+            if model_file.name in self.__validate_statuses:
+                if model_file.state in (
+                    ModelFile.State.DEFAULT,
+                    ModelFile.State.DOWNLOADED,
+                    ModelFile.State.EXTRACTED,
+                    ModelFile.State.EXTRACT_FAILED,
+                ) and model_file.local_size is not None:
+                    model_file.state = ModelFile.State.VALIDATING
+                else:
+                    self.logger.warning("File {} has validate status but is in state {}".format(
+                        model_file.name, str(model_file.state)))
```

Ruff (0.15.5)

[warning] Use f-string instead of `format` call

Convert to f-string

(UP032)